driver/config/config.go

```diff
@@ -256,7 +256,7 @@ func (s Schemas) FindSchemaByID(id string) (*Schema, error) {
 		}
 	}
 
-	return nil, errors.Errorf("could not find schema with id \"%s\"", id)
+	return nil, errors.Errorf("unable to find identity schema with id: %s", id)
```

Comment:
Example for https://github.com/ory/fosite/issues/578
This error will have a stack trace of the `init` func. Instead, we should always use `fmt.Errorf` or `import stderrs "error"`. The stack trace should instead be added once the error is actually returned somewhere.

Reply:
Oh sorry, thought this is a package level var init.

Reply:
Not sure I follow? The code style guide for Ory at the moment is to add the stack trace as "deep" as possible in the stack trace in order to get a reliable path to where the error actually happened. This works really well when tracing errors, at least in my experience. I haven't looked at ory/fosite#578

`"errors"` does to this day not include stack traces, which is why we use `pkg/errors` almost everywhere.